shared/src/main/java/uk/gov/di/authentication/entity/TICFCRIRequest.java

```diff
@@ -11,21 +12,30 @@ public record TICFCRIRequest(
         @Expose @Required List<String> vtr,
         @Expose @Required String govukSigninJourneyId,
         @Expose @Required String authenticated,
-        @Expose String initialRegistration,
-        @Expose String passwordReset) {
+        @Expose @SerializedName("initial_registration") String initialRegistration,
+        @Expose @SerializedName("password_reset") String passwordReset) {
```

Comment:
Is this change is necessary, I think we use a serialiser with the correct rules?

Reply:
I'm probably miss-understanding, but I think so?

22318995a91d416f375329aa0b83a150cbf75c6a makes a correction to `initial_registration` as it's serialised to `initialRegistration` in the `AccountInterventionsHandlerTest.java`. `AccountInterventionsHandler.java` does a gson conversion of the instance, `payload = gson.toJson(ticfRequest);`, so it seems that `AccountInterventionsHandler` was serializing incorrectly.

Reply:
Happy to leave this, but it feels weirdly inconsistent to also have `govukSigninJourneyId` when TICF CRI needs `govuk_signin_journey_id`

Reply:
I've undo this change now